evm/src/arithmetic/columns.rs

```diff
@@ -79,6 +79,11 @@ const AUX_REGISTER_2: Range<usize> = AUX_REGISTER_1.end..AUX_REGISTER_1.end + 2
 // the range checks.
 pub(crate) const AUX_COEFF_ABS_MAX: i64 = 1 << 20;
 
+// Each element c of MODULAR_QUO_INPUT is -2^16 <= c < 2^16;
+// this value is used as an offset so that everything is positive in
+// the range checks.
+pub(crate) const QUO_INPUT_ABS_MAX: i64 = 1 << 16;
```

Comment:
Nit: it feels like an odd choice to put this constant in the `columns` file. The convention we've used is that `columns` contains the indices assigned to particular columns, and this value is not an index.